### PR TITLE
Adding release notes automation to the IG

### LINKED ISSUE #35: Add automated release notes
Add automated releases tied to issues on the project board
- Releases tied to issues
- Release notes generated when release is created

TODOS
- [x] Create automated release notes for issues tied to the issue tags
- [x] Include build artifacts on tagged release (source code, snapshot jar, full-ig.zip, package.tgz)
- [x] Add notes about gren release notes to history.md, which shows up on the IG in the history.html page (linked from QA report page)

### LINKED ISSUE #38: Fix broken xml examples link on download page
The link to the examples xml file is broken, has wrong file name.

### LINKED ISSUE #39: Add releases link to IG
Releases page to be included in the IG:
- [x] Add release notes page link to README.md
- [x] Add release notes page link to IG index page 
- [x] Fix link on QA page (history)

### LINKED ISSUE #40: Remove version from canonical link
Canonical urls in the IG include the version number.  See package.json (root folder).

This is not correct, and I think we should remove it.  
- [x] Discuss with team
- [x] Remove the version

### LINKED ISSUE #43: Add license to IG home page
Update the home page to include the same licensing text that is on the README.md page

### LINKED ISSUE #44: Add versioning notes to README.md
Update readme page to include information about release version numbering

- New major versions on breaking change
     - New required fields on existing profiles
    - Tighter cardinality restrictions on existing fields
    - Tighter data type or valueset binding constraints on existing profiles 
    - Code changes to existing vocabularies
    - Url changes to existing profiles, vocabularies, etc.
     
- New minor version on non-breaking IG changes
    - New profiles or extensions
    - New codes, coding system, or vocabularies
    - Looser cardinality restrictions on existing fields
    - Looser data type or valueset binding constraints on existing profiles 
    - New optional fields on existing profiles
 
- New revision on documentation changes that are reflected in the IG, including:
    - Changes to descriptions of profiles, extensions, or codes
    - Changes on IG pages that do not impact the profiles
   
- No version bump on other repo changes that do not affect the IG
    - Documentation in github pages only that are reflected in the IG (readme.md, etc)
    - Build CI changes

### PR DESCRIPTION
**Pull request by coxstuart**
Adding release notes automation

- [x] Add automated release notes and history.html page generation when release is created
- [x] Add release notes link to readme
- [x] Fix broken xml examples link on download page
- [x] Remove version from canonical link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +39,4 @@ IBM and the IBM logo are trademarks of International Business Machines Corporati
 
 ## Release History
 
-The following Alvearie Implementation Guides have been published:
-
-| Date           | IG Version      | FHIR Version | Description |
-| -------------- | --------------- | ------------ | ----------- |
-|  7/23/2021    | [0.1.1](https://alvearie.github.io/alvearie-fhir-ig/)           | 4.0.1        | Release       |
+View the [Release][https://github.com/Alvearie/alvearie-fhir-ig/releases]. Each release includes the full IG specication.
```
Comment (dmangum2) [inline README.md]: "specication" typo.

**Review by dmangum2 — CHANGES_REQUESTED** [review]
One change requested.

File: README.md (hunk 1 above)
Comment (dmangum2) [reply README.md]: Also on the readme.md, looks strange.   image

Comment (coxstuart) [reply README.md]: Thank you.  Fixed the typos and added some more content.

**Comment by coxstuart** [comment]
> Issue labels "will" create? Or "are created on ...."

@dmangum2:  Thank you!  I fixed multiple typos and rewrote a bunch of the readme.  Please take another look.  I need "gramerly for git". Do they make that?

**Review by dmangum2 — APPROVED** [review]
Looks good!

**Comment by kimkenna** [comment]
Two typos in the readme I believe:
1. Line 37 is missing an 's' for the word Issues in this line: "* Issue are used to document changes included in the release notes for each release"
2. Line 55 is missing the preceding e for export in this line: "xport GREN_GITHUB_TOKEN=<YOUR TOKEN HERE>"

**Comment by coxstuart** [comment]
> Two typos in the readme I believe:
> 
> 1. Line 37 is missing an 's' for the word Issues in this line: "* Issue are used to document changes included in the release notes for each release"
> 2. Line 55 is missing the preceding e for export in this line: "xport GREN_GITHUB_TOKEN="

Thanks @kimkenna.  Sharp eyes!  Fixed the readme.

File: README.md [hunk 2]
```diff
@@ -39,8 +94,43 @@ IBM and the IBM logo are trademarks of International Business Machines Corporati
...
+- Zip fies containing definitions and examples in json, xml, and ttl (Turtle)
+
+## Versioning IG Releases
+
+Please follow the guidelines listed below when making changes to this repo and labeling IG releases.
 
-| Date           | IG Version      | FHIR Version | Description |
-| -------------- | --------------- | ------------ | ----------- |
-|  7/23/2021    | [0.1.1](https://alvearie.github.io/alvearie-fhir-ig/)           | 4.0.1        | Release       |
+Changes must be made to the following files to version the IG. This will ensure the automatmed releases work correctly.
```
Comment (dmangum2) [inline README.md]: automatmed misspelled

File: README.md [hunk 3]
```diff
@@ -39,8 +94,43 @@ IBM and the IBM logo are trademarks of International Business Machines Corporati
...
+
+- package.json (root folder)
+- data/package.json
+- data/config.yaml
+
+IG releases and version numbers should be modified as follows:
+
+1. **New major versions on breaking changes**
```
Comment (dmangum2) [inline README.md]: Should we note the version number scheme as N.x.x, so it's clear what pattern we are going to use?

File: README.md [hunk 4]
```diff
@@ -39,8 +94,43 @@ IBM and the IBM logo are trademarks of International Business Machines Corporati
...
+    - New required fields on existing profiles
+    - Tighter cardinality restrictions on existing fields
+    - Tighter data type or valueset binding constraints on existing profiles
+    - Code changes to existing vocabularies
+    - Url changes to existing profiles, vocabularies, etc.
+
+2. **New minor version on non-breaking IG profile changes**
```
Comment (dmangum2) [inline README.md]: Note the version number pattern, like x.N.x?

File: README.md [hunk 5]
```diff
@@ -39,8 +94,43 @@ IBM and the IBM logo are trademarks of International Business Machines Corporati
...
+    - New profiles or extensions
+    - New codes, coding system, or vocabularies
+    - Looser cardinality restrictions on existing fields
+    - Looser data type or valueset binding constraints on existing profiles
+    - New optional fields on existing profiles
+  
+3. **New revision on documentation changes that are reflected in the IG**
```
Comment (dmangum2) [inline README.md]: Note the version number pattern, like x.x.N?

**Review by dmangum2 — CHANGES_REQUESTED** [review]
I had a few comments.

File: README.md (hunk 2 above)
Comment (coxstuart) [reply README.md]: Fixed. Thank you.

File: README.md (hunk 3 above)
Comment (coxstuart) [reply README.md]: Good suggestion.  I added that.  Please review and let me know if you think we need more.

File: README.md (hunk 4 above)
Comment (coxstuart) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (coxstuart) [reply README.md]: Done

**Review by dmangum2 — APPROVED** [review]

**Review by kimkenna — APPROVED** [review]
Great work!

**Pull request merged**